test/Transport/SendmailTest.php

```diff
@@ -223,4 +223,38 @@ public function testTrimmedParameters()
 
         $this->assertSame('-R hdrs', $r->getValue($this->transport));
     }
+
+    public function testAllowMessageWithEmptyToHeaderButHasCcHeader()
+    {
+        $message = new Message();
+        $message->addCc('matthew@zend.com')
+                ->setSender('ralph.schindler@zend.com', 'Ralph Schindler')
+                ->setSubject('Testing Zend\Mail\Transport\Sendmail')
+                ->setBody('This is only a test.');
+
+        $this->transport->send($message);
+    }
+
+    public function testAllowMessageWithEmptyToHeaderButHasBccHeader()
+    {
+        $message = new Message();
+        $message->addBcc('zf-crteam@lists.zend.com', 'CR-Team, ZF Project')
+                ->setSender('ralph.schindler@zend.com', 'Ralph Schindler')
+                ->setSubject('Testing Zend\Mail\Transport\Sendmail')
+                ->setBody('This is only a test.');
+
+        $this->transport->send($message);
+    }
+
+    public function testDonotAllowMessageDoesnotHasToAndCcAndBccHeader()
+    {
+        $this->expectException('Zend\Mail\Exception\RuntimeException');
```

Comment:
Please move this before line which should thrown an exception. Like that it's confusing, and the exception could be thrown anywhere below. Can we also use `::class` notation, please?

Reply:
done